src/main/sensors/battery.c

```diff
@@ -187,7 +190,14 @@ void batteryUpdatePresence(void)
         }
 
         consumptionState = voltageState = BATTERY_OK;
-        batteryCellCount = cells;
+        if (batteryConfig()->setBatteryCellCount != 0)
```

Comment:
(`batteryCellCount = batteryConfig()->setBatteryCellCount ?  batteryConfig()->setBatteryCellCount  : cells;

Reply:
I'm not sure where/what I should place this code in the file.